rest-api/src/main/java/gaffer/rest/service/SimpleOperationService.java

```diff
@@ -79,39 +82,51 @@ public void addElements(final AddElements operation) {
     }
 
     @Override
-    public Iterable<Element> getElementsBySeed(final GetElementsSeed<ElementSeed, Element> operation) {
-        return execute(operation);
+    public Iterable<Element> getElementsBySeed(final GetElementsSeed<ElementSeed, Element> operation, final Integer n) {
+        Iterable<Element> elements = execute(operation);
```

Comment:
Done in the executeGet method to avoid having to duplicate the tertiary operator on every Call.